set.mm

```diff
@@ -29915,11 +29915,25 @@ choice between (what we call) a "definitional form" where the shorter
   $}
 
   ${
-    $d x A $.  $d y A $.  $d y ph $.  $d x ps $.
+    $d x y A $.  $d y ph $.  $d x ps $.
     cbvrabv.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
     $( Rule to change the bound variable in a restricted class abstraction,
-       using implicit substitution.  (Contributed by NM, 26-May-1999.) $)
+       using implicit substitution.  (Contributed by NM, 26-May-1999.)  Require
+       ` x ` , ` y ` be disjoint to avoid ~ ax-11 and ~ ax-13 .  (Revised by
+       Steven Nguyen, 4-Dec-2022.) $)
     cbvrabv $p |- { x e. A | ph } = { y e. A | ps } $=
+      ( cv wcel wa cab crab weq eleq1w anbi12d cbvabv df-rab 3eqtr4i ) CGEHZAIZ
```

Comment:
From about 200 applications of this theorem not a single one complains about the extra dv condition.  Interesting